driver/razerkbd_driver.c

```diff
@@ -1766,6 +1884,30 @@ static int razer_raw_event(struct hid_device *hdev, struct hid_report *report, u
     return 0;
 }
 
+/**
+ * Set static hid-events translation map
+ *
+ * These keyboards Generates wheel-events for volume control spinner
+ * - BlackWidow Elite
+ * - Huntsman Elite
```

Comment:
Please make the comment generic because the comment probably won't be updated with new devices

Reply:
+1, Thank you for your attention